clients/drcachesim/tracer/raw2trace.h

```diff
@@ -827,6 +828,11 @@ template <typename T> class trace_converter_t {
                     buf, (trace_marker_type_t)in_entry->extended.valueB, marker_val);
                 if (in_entry->extended.valueB == TRACE_MARKER_TYPE_KERNEL_EVENT) {
                     impl()->log(4, "Signal/exception between bbs\n");
+                } else if (in_entry->extended.valueB ==
+                           TRACE_MARKER_TYPE_FILTER_ENDPOINT) {
+                    impl()->log(2, "Reached filter endpoint\n");
+                    saw_filter_endpoint = true;
+                    *flush_decode_cache = true;
```

Comment:
Why does the filter endpoint require flushing decode cache? Add a comment.

Reply:
See https://github.com/DynamoRIO/dynamorio/pull/5717#discussion_r1049374210